lib/elixir/lib/kernel.ex

```diff
@@ -3401,6 +3401,45 @@ defmodule Kernel do
   In the example above, a `sum/2` function is defined; this function receives
   two arguments and returns their sum.
 
+  ## Default arguments
+
+  `\\` is used to specify a default value for a parameter of a function. For
+  example:
+
+      defmodule Foo do
+        def mul_by(x, n \\ 2) do
+          x * n
+        end
+      end
+
+      Foo.mul_by 4, 3 #=> 12
+      Foo.mul_by 4    #=> 8
+
+  The compiler translates this into multiple function clauses with different
+  arities, here `Foo.mul_by/1` and `Foo.mul_by/2`, that represent cases when
+  arguments for parameters with default values are passed or not passed.
+
+  When defining a function with default arguments as well as multiple
+  explicitly declared clauses, you must write a function head that declares the
+  defaults. For example:
+
+      defmodule Concat do
+        def join(a, b \\ nil, sep \\ " ")
```

Comment:
Let's use more descriptive names here as well. I would call the module `MyString` and the arguments `string1, string2, separator`. I know we use `a` and `b` in the example above but we're trying to move away from single-letter and cryptic variables.